src/stories/containers/ActorsAbout/ActorSummary/ActorTitleWithDescription.tsx

```diff
@@ -0,0 +1,113 @@
+import styled from '@emotion/styled';
+import { Typography, useMediaQuery } from '@mui/material';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+import ActorTitleAbout from './ActorTitleAbout';
+import type { EcosystemActor } from '@ses/core/models/dto/teamsDTO';
+
+interface Props {
+  hiddenTextDescription: boolean;
+  actorAbout: EcosystemActor;
+  showDescription: boolean;
+}
+
+export const ActorTitleWithDescription: React.FC<Props> = ({ hiddenTextDescription, actorAbout, showDescription }) => {
+  const { isLight } = useThemeContext();
+  const phone = useMediaQuery(lightTheme.breakpoints.between('table_375', 'table_834'));
+  const lessThanPhone = useMediaQuery(lightTheme.breakpoints.down('table_375'));
+  return (
+    <ContainerTitle hiddenTextDescription={hiddenTextDescription}>
+      <ActorTitleAbout actorAbout={actorAbout} hiddenTextDescription={hiddenTextDescription} />
+      {showDescription && actorAbout?.sentenceDescription && (
+        <SummaryDescription hiddenTextDescription={lessThanPhone || phone || hiddenTextDescription}>
```

Comment:
if `lessThanPhone` and `phone` are only needed for this condition, it can be modified to:

```
const isPhone = useMediaQuery(lightTheme.breakpoints.down('table_834'));
```

Reply:
done